src/components/Select/Select.stories.tsx

```diff
@@ -165,18 +165,76 @@ const selectCat: StoryObj['play'] = async (playOptions) => {
 };
 
 export const Default: StoryObj = {
+  parameters: {
+    docs: {
+      source: {
+        code: `
+// props: {aria-label: 'Favorite Animal'}
+function InteractiveExampleUsingChildren(props: Props) {
+  const { variant, optionsAlign, optionsClassName, disabled, value } = props;
+  const compact = variant === 'compact';
+
+  const [selectedOption, setSelectedOption] = React.useState<
+    SelectOption | undefined
+  >(value);
+
+  return (
+    <div className="mb-10 p-8">
+      <Select
+        aria-label={props['aria-label']}
+        className={clsx(!compact && 'w-60')}
+        data-testid="dropdown"
+        disabled={disabled}
+        onChange={setSelectedOption}
+        optionsAlign={optionsAlign}
+        optionsClassName={optionsClassName}
+        value={selectedOption}
+        variant={variant}
+      >
+        {props.labelComponent}
+        <Select.Button>{selectedOption?.label || 'Select'}</Select.Button>
+        <Select.Options>
+          {exampleOptions.map((option) => (
+            <Select.Option key={option.key} value={option}>
+              {option.label}
+            </Select.Option>
+          ))}
+        </Select.Options>
+      </Select>
+    </div>
+  );
+}`,
+      },
+    },
+  },
   render: () => (
     <InteractiveExampleUsingChildren aria-label="Favorite Animal" />
   ),
 };
 
 export const Disabled: StoryObj = {
+  parameters: {
+    docs: {
+      source: {
+        code: '// refer to "Default" story code',
+      },
+    },
+  },
```

Comment:
I got lazy with some of these but didn't want to explicit state props here if the story props were to change in the future, then would have to remember to change here as well... also the code example should be enough